compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -96,11 +97,13 @@ LogicalResult setMatmulLoweringConfig(IREE::GPU::TargetAttr target,
     // and a larger bestKTileCountPerSubgroup.
     seeds = {/*bestSubgroupCountPerWorkgroup=*/4,
              /*bestMNTileCountPerSubgroup=*/4,
-             /*bestKTileCountPerSubgroup=*/8};
+             /*bestKTileCountPerSubgroup=*/8,
+             /*bestKElementCountPerSubgroup*/ 512 / inBitWidth};
```

Comment:
Could we spell out where this comes from? I'd expect this to be `cacheLineSizeBits / inBitWidth`, so `1024 / inBitWidth`

Reply:
Is it 512 because we need to load both the LHS and the RHS?

Reply:
Maybe it would be less confusing if we could say what is the number of K elements across all of the K tiles?

Reply:
This came from some manual tuning, but I tried 1024 and it is an improvement! I'll change it and add some descriptive naming.